### ISSUE TITLE
No longer works?

### ISSUE THREAD

**Issue by loshjawrence** [akinsho/nvim-bufferline.lua#78]
Recently did a `:PlugUpdate` and buffer tabs were not showing up.

my minimal .vimrc (.vimrc-test). I launch with `nvim -u ~/.vimrc-test`
```
call plug#begin("~/.vim" . '/bundle')
Plug 'akinsho/nvim-bufferline.lua'
call plug#end()
```

Result of `nvim --version`
```
NVIM v0.5.0-dev+1229-g91cdc1198
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/loshjawrence/clones/neovim/build/config -I/home/loshjawrence/clones/neovim/src -I/home/loshjawrence/clones/neovim/.deps/usr/include -I/usr/include -I/home/loshjawrence/clones/neovim/build/src/nvim/auto -I/home/loshjawrence/clones/neovim/build/include
Compiled by loshjawrence@Lappy486

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

Also when I try to run with my normal .vimrc that has this configuration
```
" nvim-bufferline
:lua << EOF
  local hlColor = "GreenYellow"
  require'bufferline'.setup{
    -- override some options from their defaults
    options = {
        tab_size = 12,
        max_name_length = 40,
        show_buffer_close_icons = false,
    },
     highlights = {
         buffer_selected = {
             guifg = "Black",
             guibg = hlColor,
             gui = "bold",
         },
         -- Accent the split buffer thats not selected
         buffer_visible = {
             guifg = hlColor,
         },
     },
  }
EOF

```

I get this error on startup:
![image](https://user-images.githubusercontent.com/13598291/113868121-52282180-977d-11eb-87e5-df23c2fc6131.png)

**Comment by akinsho** [comment]
@loshjawrence try `PlugUpdate` again I've already pushed a fix for this #77

**Comment by loshjawrence** [comment]
@akinsho Fixed, thanks!
